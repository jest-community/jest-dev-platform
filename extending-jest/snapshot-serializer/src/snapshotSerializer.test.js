expect.addSnapshotSerializer({
  // TODO: implement me
});

const movies = [
  { year: 2013, title: 'Frozen', emoji: '❄️👗'},
  { year: 'TBA', title: 'Frozen 2', emoji: '️️️️❄️👗✌️' },
  { year: 2003, title: 'Finding Nemo', emoji: '🔎🐠' },
  { year: 2008, title: 'Kung Fu Panda', emoji: '👊💥🐼' }
];

const searchMovies = (char) => movies.filter(movie => movie.title.includes(char));

it('Returns the movies that match the query', () => {
  expect(searchMovies('F')).toMatchSnapshot();
});
