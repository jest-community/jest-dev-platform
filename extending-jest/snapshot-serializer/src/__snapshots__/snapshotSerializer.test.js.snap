// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Returns the movies that match the query 1`] = `
Array [
  ❄️👗 Frozen,
  ️️️️❄️👗✌️ Frozen 2,
  🔎🐠 Finding Nemo,
  👊💥🐼 Kung Fu Panda,
]
`;
